zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/BasicAuthInterceptor.java

```diff
@@ -19,28 +19,28 @@
 import com.linecorp.armeria.common.HttpHeaderNames;
 import com.linecorp.armeria.common.HttpRequest;
 import com.linecorp.armeria.common.HttpResponse;
-import java.util.Base64;
-
-import static java.nio.charset.StandardCharsets.UTF_8;
+import java.util.Optional;
 
 /**
- * Adds basic auth username and password to every request per https://www.elastic.co/guide/en/x-pack/current/how-security-works.html
+ * Adds basic auth username and password to every request per.
```

Comment:
`... to every request per.` incomplete sentence ?

Reply:
drop "per"